### DISCUSSION TITLE
Edit LIST menu item when keyboard not have UP/DOWN buttons

### DISCUSSION THREAD

**Discussion by rozrabiak** [forntoh/LcdMenu#58] [Q&A] [answered]
My keyboard have only 5 buttons.
LEFT, RIGHT, POWER, AUTO, MENU.
When I press MENU - MENU will open.
LEFT/RIGHT I use like UP/DOWN through menu. But I need to use LEFT/RIGHT to change values on LIST.
How can I check if menu item is list and then I invoke in code menu.right() or menu.left() to change values on list?
And after I exit from menu item LIST LEFT/RIGHT will work to change menu items..
Help?

**Comment by forntoh** [comment, marked as answer]
This is not an issue with the library, the library just provides you with everything you need to build any type of menu you want. Moving it to Discussions 

To answer your question: The library exposes utility methods that will help you know the type of the current menu item.
See [menu controls](https://lcdmenu.forntoh.dev/controls), then have a look at the accessible properties of `MenuItem`
